docs/tutorials/install_and_admin/geonode_install/install_geoserver_application.txt

```diff
@@ -1,110 +1,103 @@
 .. _install_geoserver_application:
 
-=============================
-Install GeoServer Application
-=============================
+=================
+Install GeoServer
+=================
 
 In this section we are going to setup GeoServer for GeoNode. GeoServer will run inside
-`Tomcat` sevrlet container.
+`Tomcat` servlet container.
 
-Setup GeoServer
-===============
+Download and Setup
+==================
+
+#. First download GeoServer by help of Paver. (Make sure your virtuualenv is activated!)
+
+    .. code-block:: bash
 
-#. You've already installed `Tomcat 8` in the system in the first section of the training. Before you deploy GeoServer stop the running Tomcat instance
+        source /home/geonode/.venv/my_geonode/bin/activate
+        cd /home/geonode/my_geonode/ 
+        paver setup
+
+#. You've already installed `Tomcat 8` in the first section of the training. Before you deploy GeoServer stop the running Tomcat instance.
 
     .. code-block:: bash
 
-        $ sudo service tomcat8 stop
+        sudo service tomcat8 stop
 
 #. Now copy the downloaded GeoServer archive inside Tomcat's webapps folder
 
     .. code-block:: bash
 
-        $ sudo cp -Rf /home/geonode/my_geonode/geoserver/geoserver/ /var/lib/tomcat8/webapps/
+        sudo cp /home/geonode/my_geonode/downloaded/geoserver-2.*.war /var/lib/tomcat8/webapps/geoserver.war
 
-#. Move ``GEOSERVER_DATA_DIR`` on an external location
+#. And update catalina.properties
 
     .. code-block:: bash
 
-        $ sudo mkdir -p /data/geoserver-data
-        $ sudo mkdir -p /data/geoserver-logs
-        $ sudo mkdir -p /data/gwc_cache_dir
-        $ sudo cp -Rf /home/geonode/my_geonode/geoserver/data/* /data/geoserver-data/
-        $ sudo chown -Rf tomcat8: /data/geoserver-data/
-        $ sudo chown -Rf tomcat8: /data/geoserver-logs/
-        $ sudo chown -Rf tomcat8: /data/gwc_cache_dir/
+        sudo sed -i -e 's/xom-\*\.jar/xom-\*\.jar,bcprov\*\.jar/g' /var/lib/tomcat8/conf/catalina.properties
+
+
+
+#. To keep things tidied move ``GEOSERVER_DATA_DIR`` to an external location
+
+    .. code-block:: bash
+
+        sudo mkdir -p /data/geoserver-data;
+        sudo mkdir -p /data/geoserver-logs;
+        sudo mkdir -p /data/gwc_cache_dir;
+        sudo cp -Rf /home/geonode/my_geonode/geoserver/data/* /data/geoserver-data/;
+        sudo chown -Rf tomcat8:tomcat8 /data/geoserver-data/;
+        sudo chown -Rf tomcat8:tomcat8 /data/geoserver-logs/;
+        sudo chown -Rf tomcat8:tomcat8 /data/gwc_cache_dir/;
```

Comment:
`;` can be removed.

Reply:
The ; was meant to copy and paste the whole part into terminal (instead of line per line). But good point let´s stay with the one or the other.